api/app/signals/apps/api/v1/views/pdf.py

```diff
@@ -101,10 +151,13 @@ def get_context_data(self, **kwargs):
                 rd_coordinates.x + 340.00,
                 rd_coordinates.y + 125.00,
             )
+        jpg_data_urls = self._get_context_data_images(self.object)
+
         return super(GeneratePdfView, self).get_context_data(
             bbox=bbox,
             img_data_uri=img_data_uri,
```

Comment:
Is this still needed if we are loading the base64 encoded images?

Reply:
That function call is doing the actual resizing and encoding. The template expects `jpg_data_urls` to be present in the context if `images` is not.